### PR TITLE
[mimir-distributed] use global.clusterDomain for internal services

### PR DESCRIPTION
**Pull request by secustor**
This PR replaces usages of `cluster.local` with the existing variable `global.clusterDomain`.     

Therefore it resolves routing issues if Mimir is installed in cluster which make use unique cluster domains. 

Signed-off-by: secustor <sebastian@poxhofer.at>

### CODE REVIEW

File: charts/mimir-distributed/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-version: 2.0.0
+version: 2.1.0
```
Comment (krajorama) [inline charts/mimir-distributed/Chart.yaml:2]:
```suggestion
version: 2.0.1
```

File: charts/mimir-distributed/README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Helm chart for deploying [Grafana Mimir](https://grafana.com/docs/mimir/v2.0.x/)
 
 # mimir-distributed
 
-![Version: 2.0.0](https://img.shields.io/badge/Version-2.0.0-informational?style=flat-square) ![AppVersion: 2.0.0](https://img.shields.io/badge/AppVersion-2.0.0-informational?style=flat-square)
+![Version: 2.1.0](https://img.shields.io/badge/Version-2.1.0-informational?style=flat-square) ![AppVersion: 2.0.0](https://img.shields.io/badge/AppVersion-2.0.0-informational?style=flat-square)
```
Comment (krajorama) [inline charts/mimir-distributed/README.md:7]:
```suggestion
![Version: 2.0.1](https://img.shields.io/badge/Version-2.0.1-informational?style=flat-square) ![AppVersion: 2.0.0](https://img.shields.io/badge/AppVersion-2.0.0-informational?style=flat-square)
```

**Review by krajorama — COMMENTED** [review]
Change looks good, thank you. I think it should be version 2.0.x (x>0) depending on when we merge, because it's a bugfix, and I'd kindly ask you to update the CHANGELOG as well. (Or I can take over if you prefer from this point).

**Comment by secustor** [comment]
I have changed the Chart version to `2.0.1` and added a change log entry

**Review by krajorama — APPROVED** [review]
LGTM

**Pull request merged**